Can't link ../../bin/nrniv

PizDaint has updated their modules etc and I have to rebuild NEURON. I clone the 7.8.1 tag and run 

```
module load PrgEnv-gnu

# Clone nrn repo
git clone --depth 1 --branch 7.8.1 https://github.com/neuronsimulator/nrn

# Run CMake & install
CDIR=$PWD
mkdir ~/nrn/build
cd ~/nrn/build
cmake .. \
  -DCMAKE_INSTALL_PREFIX=$HOME/nrn-install \
  -DNRN_ENABLE_CORENEURON=OFF \
  -DNRN_ENABLE_INTERVIEWS=OFF \
  -DPYTHON_EXECUTABLE=/opt/python/3.8.2.1/bin/python

make -j
make install
```

It finds the Python executable and when I execute the Cmake steps that determine the library dir using the python interpreter (`import distutils; print(distutils.get_config_var('library_dir'))`) it correctly prints out the library folder. Cmake should know where to find the `libpython3.8.so`, still, in the build logs it reports `/usr/bin/ld can't find -lpython3.8` (why is it using ld for linking?):

```
bp000347@daint105:~> source bsb-daint/install-neuron.sh
Cloning into 'nrn'...
remote: Enumerating objects: 1948, done.
remote: Counting objects: 100% (1948/1948), done.
remote: Compressing objects: 100% (1780/1780), done.
remote: Total 1948 (delta 175), reused 1136 (delta 100), pack-reused 0
Receiving objects: 100% (1948/1948), 8.24 MiB | 10.39 MiB/s, done.
Resolving deltas: 100% (175/175), done.
Note: switching to '47cd8c85aa3fde5dbb7319facd6f475438235d7d'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

Updating files: 100% (1875/1875), done.
-- The C compiler identification is GNU 8.3.0
-- The CXX compiler identification is GNU 8.3.0
-- Cray Programming Environment 2.7.0 C
-- Check for working C compiler: /opt/cray/pe/craype/2.7.0/bin/cc
-- Check for working C compiler: /opt/cray/pe/craype/2.7.0/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Cray Programming Environment 2.7.0 CXX
-- Check for working CXX compiler: /opt/cray/pe/craype/2.7.0/bin/CC
-- Check for working CXX compiler: /opt/cray/pe/craype/2.7.0/bin/CC -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Setting build type to 'RelWithDebInfo' as none was specified.
-- Found BISON: /usr/bin/bison (found version "3.0.4")
-- Found FLEX: /usr/bin/flex (found version "2.6.4")
-- Found PythonInterp: /opt/python/3.8.2.1/bin/python (found version "3.8.2")
-- Found Cython: /apps/dom/UES/jenkins/7.0.UP02/mc/easybuild/software/PyExtensions/python3-CrayGNU-20.08/bin/cython (found version "0.29.21")
-- Found MPI_C: /opt/cray/pe/craype/2.7.0/bin/cc (found version "3.1")
-- Found MPI_CXX: /opt/cray/pe/craype/2.7.0/bin/CC (found version "3.1")
-- Found MPI: TRUE (found version "3.1")
-- Found PythonInterp: /opt/python/3.8.2.1/bin/python (found suitable version "3.8.2", minimum required is "3")
-- Found PythonLibs: python3.8
-- Found Threads: TRUE
CMake Warning (dev) at src/nrniv/CMakeLists.txt:254 (add_library):
  ADD_LIBRARY called with SHARED option but the target platform does not
  support dynamic linking.  Building a STATIC library instead.  This may lead
  to problems.
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Warning (dev) at src/nrnpython/CMakeLists.txt:91 (add_library):
  ADD_LIBRARY called with SHARED option but the target platform does not
  support dynamic linking.  Building a STATIC library instead.  This may lead
  to problems.
This warning is for project developers.  Use -Wno-dev to suppress it.

--
-- Configured NEURON 7.8
--
-- Some things you can do now:
-- --------------+--------------------------------------------------------------
-- Command       |   Description
-- --------------+--------------------------------------------------------------
-- make install  | Will install NEURON to: /users/bp000347/nrn-install
--               | Change the install location of NEURON using:
--               |     cmake <src_path> -DCMAKE_INSTALL_PREFIX=<install_path>
-- make docs     | Build the API documentation
-- make uninstall| Removes files installed by make install (todo)
-- --------------+--------------------------------------------------------------
--  Build option | Status
-- --------------+--------------------------------------------------------------
-- BUILD_TYPE    | RelWithDebInfo (allowed: Custom;Debug;Release;RelWithDebInfo;Fast)
-- COMPILE FLAGS | -g  -O2
-- Shared        | ON
-- Binary special| OFF
-- Legacy FR     | ON
-- MPI           | ON
--   INC         |
--   LIB         | MPI_LIBRARY-NOTFOUND
--   DYNAMIC     | OFF
-- Python        | ON
--   EXE         | /opt/python/3.8.2.1/bin/python
--   INC         | /opt/python/3.8.2.1/include/python3.8
--   LIB         | python3.8
--   MODULE      | ON
--   DYNAMIC     | OFF
-- Curses        | /usr/lib64/libncurses.a;/usr/lib64/libtinfo.a;/usr/lib64/libform.a
-- RX3D          | ON
--   OptLevel    | 0
-- Interviews    | OFF
-- CoreNEURON    | OFF
-- --------------+--------------------------------------------------------------
--  See documentation : https://www.neuron.yale.edu/neuron/
-- --------------+--------------------------------------------------------------
--
-- Configuring done
-- Generating done
-- Build files have been written to: /users/bp000347/nrn/build
Scanning dependencies of target readline
[  0%] [FLEX][nocmodllexer] Building scanner with flex 2.6.4
[  1%] [FLEX][modlunitlexer] Building scanner with flex 2.6.4
[  1%] [BISON][modlunitparser] Building parser with bison 3.0.4
[  1%] [BISON][nocmodlparser] Building parser with bison 3.0.4
[  1%] [BISON][nocmodlparser] Building parser with bison 3.0.4
Scanning dependencies of target hh_update
Scanning dependencies of target nrnversion_h
Scanning dependencies of target rxdmath
Scanning dependencies of target rxd_cython_generated
/users/bp000347/nrn/src/nmodl/diffeq.y: warning: 5 shift/reduce conflicts [-Wconflicts-sr]
/users/bp000347/nrn/src/nmodl/parse1.y: warning: 3 shift/reduce conflicts [-Wconflicts-sr]
[  1%] Update hh.mod for CoreNEURON compatibility
[  1%] Building C object src/readline/CMakeFiles/readline.dir/history.c.o
[  3%] Generating ctng.cpp
[  3%] Building C object src/readline/CMakeFiles/readline.dir/funmap.c.o
[  3%] Building C object src/readline/CMakeFiles/readline.dir/readline.c.o
[  3%] Building C object src/nrnpython/CMakeFiles/rxdmath.dir/rxdmath.c.o
[  3%] Generating surfaces.cpp
[  3%] Generating graphicsPrimitives.cpp
[  3%] Building C object src/readline/CMakeFiles/readline.dir/keymaps.c.o
[  3%] Built target hh_update
[  4%] Linking C static library ../../lib/librxdmath.a
Scanning dependencies of target modlunit
[  4%] Built target rxdmath
Scanning dependencies of target nocmodl
[  4%] Building C object src/nrniv/CMakeFiles/modlunit.dir/__/modlunit/lex.c.o
[  4%] Building C object src/nrniv/CMakeFiles/modlunit.dir/__/modlunit/parse1.c.o
[  5%] Building C object src/nrniv/CMakeFiles/modlunit.dir/__/modlunit/nrnunit.c.o
[  6%] Building C object src/nrniv/CMakeFiles/modlunit.dir/__/modlunit/init.c.o
[  6%] Building C object src/nrniv/CMakeFiles/modlunit.dir/__/modlunit/kinunit.c.o
[  6%] Building C object src/nrniv/CMakeFiles/modlunit.dir/__/modlunit/passn.c.o
[  7%] Building C object src/nrniv/CMakeFiles/modlunit.dir/__/modlunit/version.c.o
[  7%] Building C object src/nrniv/CMakeFiles/modlunit.dir/__/modlunit/model.c.o
[  7%] Building C object src/nrniv/CMakeFiles/modlunit.dir/__/modlunit/units.c.o
[  7%] Building C object src/nrniv/CMakeFiles/modlunit.dir/__/modlunit/consist.c.o
[  7%] Building C object src/nrniv/CMakeFiles/modlunit.dir/__/modlunit/io.c.o
[  7%] Building C object src/nrniv/CMakeFiles/modlunit.dir/__/modlunit/declare.c.o
[  7%] Building C object src/nrniv/CMakeFiles/modlunit.dir/__/modlunit/list.c.o
[  7%] Building C object src/nrniv/CMakeFiles/modlunit.dir/__/modlunit/symbol.c.o
[  7%] Building C object src/nrniv/CMakeFiles/modlunit.dir/__/modlunit/units1.c.o
[  8%] Building C object src/nrniv/CMakeFiles/nocmodl.dir/__/nmodl/parse1.c.o
[  8%] Building C object src/nrniv/CMakeFiles/nocmodl.dir/__/nmodl/consist.c.o
[  8%] Building C object src/nrniv/CMakeFiles/nocmodl.dir/__/nmodl/noccout.c.o
[  8%] Building C object src/nrniv/CMakeFiles/nocmodl.dir/__/nmodl/deriv.c.o
[  8%] Building C object src/nrniv/CMakeFiles/nocmodl.dir/__/nmodl/kinetic.c.o
[  8%] Building C object src/nrniv/CMakeFiles/nocmodl.dir/__/nmodl/lex.c.o
[  8%] Building C object src/nrniv/CMakeFiles/nocmodl.dir/__/nmodl/parsact.c.o
[  9%] Building C object src/nrniv/CMakeFiles/nocmodl.dir/__/nmodl/discrete.c.o
[  9%] Building C object src/nrniv/CMakeFiles/nocmodl.dir/__/nmodl/list.c.o
[  9%] Building C object src/nrniv/CMakeFiles/nocmodl.dir/__/nmodl/init.c.o
[ 10%] Building C object src/nrniv/CMakeFiles/nocmodl.dir/__/nmodl/modl.c.o
[ 10%] Building C object src/nrniv/CMakeFiles/nocmodl.dir/__/nmodl/diffeq.c.o
[ 10%] Building C object src/nrniv/CMakeFiles/nocmodl.dir/__/nmodl/netrec_discon.c.o
[ 10%] Building C object src/nrniv/CMakeFiles/nocmodl.dir/__/nmodl/simultan.c.o
[ 10%] Building C object src/nrniv/CMakeFiles/nocmodl.dir/__/nmodl/io.c.o
[ 11%] Building C object src/nrniv/CMakeFiles/nocmodl.dir/__/nmodl/partial.c.o
[ 11%] Building C object src/nrniv/CMakeFiles/nocmodl.dir/__/nmodl/solve.c.o
[ 11%] Building C object src/nrniv/CMakeFiles/nocmodl.dir/__/nmodl/symbol.c.o
[ 11%] Building C object src/nrniv/CMakeFiles/nocmodl.dir/__/nmodl/sens.c.o
[ 12%] Building C object src/nrniv/CMakeFiles/nocmodl.dir/__/nmodl/units.c.o
[ 12%] Building C object src/nrniv/CMakeFiles/nocmodl.dir/__/nmodl/nocpout.c.o
[ 12%] Building C object src/nrniv/CMakeFiles/nocmodl.dir/__/nmodl/version.c.o
[ 12%] Built target nrnversion_h
[ 12%] Linking C static library ../../lib/libreadline.a
+ file=/users/bp000347/nrn/build/share/lib/python/neuron/rxd/geometry3d/surfaces.cpp
+ mingw=
+ test x = x1
+ file=/users/bp000347/nrn/build/share/lib/python/neuron/rxd/geometry3d/ctng.cpp
+ mingw=
+ test x = x1
+ file=/users/bp000347/nrn/build/share/lib/python/neuron/rxd/geometry3d/graphicsPrimitives.cpp
+ mingw=
+ test x = x1
[ 12%] Linking C executable ../../bin/modlunit
[ 12%] Built target readline
[ 12%] Built target rxd_cython_generated
Scanning dependencies of target rx3dextensions_0
+ echo runpy /opt/python/3.8.2.1/bin/python
runpy /opt/python/3.8.2.1/bin/python
+ pyexe=/opt/python/3.8.2.1/bin/python
+ mingw=
+ shift
+ test x = x1
+ /opt/python/3.8.2.1/bin/python setup.py build_ext --build-lib=/users/bp000347/nrn/build/lib/python
/opt/cray/pe/craype/2.7.0/bin/CC
running build_ext
building 'neuron.rxd.geometry3d.graphicsPrimitives' extension
creating build
creating build/temp.linux-x86_64-3.8
/opt/cray/pe/craype/2.7.0/bin/cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -march=x86-64 -march=x86-64 -fPIC -I/users/bp000347/nrn/share/lib/python/neuron/rxd/geometry3d -I. -I/opt/python/3.8.2.1/lib/python3.8/site-packages/numpy/core/include -I/opt/python/3.8.2.1/include/python3.8 -c graphicsPrimitives.cpp -o build/temp.linux-x86_64-3.8/graphicsPrimitives.o -O0
[ 12%] Built target modlunit
[ 12%] Linking C executable ../../bin/nocmodl
[ 12%] Built target nocmodl
[ 12%] [BISON][ocparser] Building parser with bison 3.0.4
[ 12%] Generating ../nrnoc/hocusr.h
[ 14%] Generating ../../../src/nrnoc/syn.c
[ 14%] Generating ../../../src/nrnoc/expsyn.c
[ 14%] Generating ../../../src/nrnoc/intfire4.c
[ 14%] Generating ../../../src/nrnoc/netstim.c
[ 15%] Generating ../../../src/nrnoc/passive.c
[ 15%] Generating ../../../src/nrnoc/vclmp.c
[ 16%] Generating ../../../src/nrnoc/hh.c
[ 16%] Generating ../../../src/nrnoc/ppmark.c
[ 16%] Generating ../../../src/nrnoc/intfire2.c
[ 16%] Generating ../../../src/nrnoc/svclmp.c
[ 16%] Generating ../../../src/nrnoc/intfire1.c
[ 16%] Generating ../../../src/nrnoc/pattern.c
[ 16%] Generating ../../../src/nrnoc/exp2syn.c
[ 16%] Generating ../../../src/nrnoc/feature.c
[ 16%] Generating ../../../src/nrnoc/oclmp.c
[ 15%] Generating ../../../src/nrnoc/apcount.c
[ 16%] Generating ../../../src/nrnoc/stim.c
/users/bp000347/nrn/src/oc/parse.y: warning: 98 shift/reduce conflicts [-Wconflicts-sr]
Translating /users/bp000347/nrn/src/nrnoc/syn.mod into /users/bp000347/nrn/src/nrnoc/syn.c
Thread Safe
Translating /users/bp000347/nrn/src/nrnoc/exp2syn.mod into /users/bp000347/nrn/src/nrnoc/exp2syn.c
Translating /users/bp000347/nrn/src/nrnoc/passive.mod into /users/bp000347/nrn/src/nrnoc/passive.c
Thread Safe
Thread Safe
Translating /users/bp000347/nrn/src/nrnoc/stim.mod into /users/bp000347/nrn/src/nrnoc/stim.c
Translating /users/bp000347/nrn/src/nrnoc/svclmp.mod into /users/bp000347/nrn/src/nrnoc/svclmp.c
Translating /users/bp000347/nrn/src/nrnoc/apcount.mod into /users/bp000347/nrn/src/nrnoc/apcount.c
Translating /users/bp000347/nrn/src/nrnoc/vclmp.mod into /users/bp000347/nrn/src/nrnoc/vclmp.c
Translating /users/bp000347/nrn/src/nrnoc/ppmark.mod into /users/bp000347/nrn/src/nrnoc/ppmark.c
Notice: VERBATIM blocks are not thread safe
Thread Safe
Notice: LINEAR is not thread safe.
Thread Safe
Translating /users/bp000347/nrn/src/nrnoc/oclmp.mod into /users/bp000347/nrn/src/nrnoc/oclmp.c
Thread Safe
Thread Safe
Translating /users/bp000347/nrn/src/nrnoc/feature.mod into /users/bp000347/nrn/src/nrnoc/feature.c
Notice: VERBATIM blocks are not thread safe
Thread Safe
Translating /users/bp000347/nrn/src/nrnoc/expsyn.mod into /users/bp000347/nrn/src/nrnoc/expsyn.c
Translating /users/bp000347/nrn/src/nrnoc/intfire4.mod into /users/bp000347/nrn/src/nrnoc/intfire4.c
Translating /users/bp000347/nrn/src/nrnoc/hh.mod into /users/bp000347/nrn/src/nrnoc/hh.c
Translating /users/bp000347/nrn/src/nrnoc/intfire2.mod into /users/bp000347/nrn/src/nrnoc/intfire2.c
Translating /users/bp000347/nrn/src/nrnoc/intfire1.mod into /users/bp000347/nrn/src/nrnoc/intfire1.c
Thread Safe
Thread Safe
Thread Safe
Thread Safe
Thread Safe
Translating /users/bp000347/nrn/src/nrnoc/netstim.mod into /users/bp000347/nrn/src/nrnoc/netstim.c
Notice: ARTIFICIAL_CELL is a synonym for POINT_PROCESS which hints that it
only affects and is affected by discrete events. As such it is not
located in a section and is not associated with an integrator
Thread Safe
Translating /users/bp000347/nrn/src/nrnoc/pattern.mod into /users/bp000347/nrn/src/nrnoc/pattern.c
Notice: Use of POINTER is not thread safe.
Notice: VERBATIM blocks are not thread safe
Notice: ARTIFICIAL_CELL is a synonym for POINT_PROCESS which hints that it
only affects and is affected by discrete events. As such it is not
located in a section and is not associated with an integrator
    Scanning dependencies of target nrniv_lib
creating /users/bp000347/nrn/build/lib/python
creating /users/bp000347/nrn/build/lib/python/neuron
creating /users/bp000347/nrn/build/lib/python/neuron/rxd
creating /users/bp000347/nrn/build/lib/python/neuron/rxd/geometry3d
/opt/cray/pe/craype/2.7.0/bin/CC -pthread -shared -Wl,-rpath,/opt/python/3.8.2.1/lib build/temp.linux-x86_64-3.8/graphicsPrimitives.o -L/opt/python/3.8.2.1/lib -o /users/bp000347/nrn/build/lib/python/neuron/rxd/geometry3d/graphicsPrimitives.cpython-38-x86_64-linux-gnu.so
building 'neuron.rxd.geometry3d.ctng' extension
/opt/cray/pe/craype/2.7.0/bin/cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -march=x86-64 -march=x86-64 -fPIC -I/users/bp000347/nrn/share/lib/python/neuron/rxd/geometry3d -I. -I/opt/python/3.8.2.1/lib/python3.8/site-packages/numpy/core/include -I/opt/python/3.8.2.1/include/python3.8 -c ctng.cpp -o build/temp.linux-x86_64-3.8/ctng.o -O0
[ 16%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/checkpnt.cpp.o
[ 16%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/grglyph.cpp.o
[ 17%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/cbwidget.cpp.o
[ 17%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/hocmark.cpp.o
[ 17%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/grmanip.cpp.o
[ 17%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/fourier.cpp.o
[ 17%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/graphvec.cpp.o
[ 17%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/epsprint.cpp.o
[ 17%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/axis.cpp.o
[ 18%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/bndedval.cpp.o
[ 18%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/graph.cpp.o
[ 18%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/apwindow.cpp.o
[ 18%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/ivoc.cpp.o
[ 18%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/idraw.cpp.o
[ 19%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/htlist.cpp.o
[ 19%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/gifimage.cpp.o
[ 19%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/mymath.cpp.o
[ 20%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/ivocvect.cpp.o
[ 20%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/mlinedit.cpp.o
[ 20%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/objcmd.cpp.o
[ 20%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/ivocrand.cpp.o
[ 20%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/ivocmain.cpp.o
[ 21%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/oc2iv.cpp.o
[ 21%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/ocbox.cpp.o
[ 22%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/ocpicker.cpp.o
[ 22%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/ocdeck.cpp.o
[ 22%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/matrix.cpp.o
[ 22%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/ocbrowsr.cpp.o
[ 22%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/ocobserv.cpp.o
[ 23%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/symchoos.cpp.o
[ 23%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/cvodestb.cpp.o
[ 23%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/cvodeobj.cpp.o
[ 23%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/rubband.cpp.o
[ 23%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/xyview.cpp.o
[ 23%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/bbsrcli.cpp.o
[ 23%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/ocmatrix.cpp.o
[ 23%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/cvtrset.cpp.o
[ 23%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/ocpointer.cpp.o
[ 23%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/classreg.cpp.o
[ 23%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/ocnoiv1.cpp.o
[ 23%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/strfun.cpp.o
[ 24%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/ocfile.cpp.o
[ 24%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/oclist.cpp.o
[ 25%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/bbslsrv.cpp.o
[ 25%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/bbssrv.cpp.o
[ 25%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/bbslsrv2.cpp.o
[ 25%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/ochelp.cpp.o
[ 25%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/finithnd.cpp.o
[ 25%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/octimer.cpp.o
[ 25%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/geometry3d.cpp.o
[ 26%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/pwman.cpp.o
[ 26%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/field.cpp.o
[ 26%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/cxprop.cpp.o
[ 26%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/bgpmeminfo.c.o
[ 26%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/utility.cpp.o
[ 26%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/bbslocal.cpp.o
[ 26%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/matrixmap.cpp.o
[ 27%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/rect.cpp.o
[ 27%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/multisplit.cpp.o
[ 28%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/xdep.cpp.o
[ 29%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/scene.cpp.o
[ 29%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/cachevec.cpp.o
[ 29%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/xmenu.cpp.o
[ 29%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/datapath.cpp.o
[ 29%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/nrnpy.cpp.o
[ 29%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/bbsdirect.cpp.o
[ 29%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/impedanc.cpp.o
[ 29%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/bbs.cpp.o
[ 29%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/nonlinz.cpp.o
[ 29%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/glinerec.cpp.o
[ 29%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/bbsavestate.cpp.o
[ 29%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/netcvode.cpp.o
[ 29%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/scenepic.cpp.o
[ 29%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivoc/ocptrvector.cpp.o
[ 30%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/hocmech.cpp.o
[ 30%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/nrnbbcore_write.cpp.o
[ 30%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/nrndaspk.cpp.o
[ 31%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/linmod1.cpp.o
[ 31%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/nrnste.cpp.o
[ 31%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/ndatclas.cpp.o
[ 31%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/kssingle.cpp.o
[ 31%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/linmod.cpp.o
[ 31%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/nvector_nrnthread_ld.c.o
[ 31%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/nvector_nrnserial_ld.c.o
[ 31%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/ocjump.cpp.o
[ 31%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/kschan.cpp.o
[ 32%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/nrnrtime.cpp.o
[ 32%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/netpar.cpp.o
[ 32%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/ocbbs.cpp.o
[ 33%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/nvector_nrnthread.c.o
[ 34%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/nrndae.cpp.o
[ 34%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/nrnmenu.cpp.o
[ 34%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/pysecname2sec.cpp.o
[ 35%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/partrans.cpp.o
[ 35%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/prcellstate.cpp.o
[ 36%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/rotate3d.cpp.o
[ 36%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/occvode.cpp.o
[ 36%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/secbrows.cpp.o
[ 37%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/eion.c.o
[ 37%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/tqueue.cpp.o
[ 38%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/singlech.cpp.o
[ 38%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/extcelln.c.o
[ 38%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/shape.cpp.o
[ 38%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/clamp.c.o
[ 38%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/cabcode.c.o
[ 38%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/shapeplt.cpp.o
[ 38%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/symdir.cpp.o
[ 38%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/hocprax.c.o
[ 38%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/code.c.o
[ 39%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/ppshape.cpp.o
[ 39%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/vrecord.cpp.o
[ 39%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/savstate.cpp.o
[ 39%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/fstim.c.o
[ 39%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/capac.c.o
[ 39%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/spaceplt.cpp.o
[ 39%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/splitcell.cpp.o
[ 39%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/hoc_oop.c.o
[ 39%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/nrntimeout.c.o
[ 39%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/fadvance.c.o
[ 39%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/ldifus.c.o
[ 39%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/hoc_init.c.o
[ 41%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/method3.c.o
[ 41%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/init.c.o
[ 41%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/hocusr.c.o
[ 42%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/synapse.c.o
[ 42%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/nrnversion.c.o
[ 42%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/psection.c.o
[ 42%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/seclist.c.o
[ 42%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/oc/audit.c.o
[ 42%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/nrnnemo.c.o
[ 42%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/solve.c.o
[ 42%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/oc/debug.c.o
[ 42%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/oc/nrnisaac.c.o
[ 42%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/passive0.c.o
[ 42%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/oc/isaac64.c.o
[ 43%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/point.c.o
[ 43%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/secref.c.o
[ 43%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/oc/mcran4.c.o
[ 43%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/treeset.c.o
[ 44%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/oc/cygwinprt.c.o
[ 44%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/oc/axis.c.o
In file included from /opt/python/3.8.2.1/lib/python3.8/site-packages/numpy/core/include/numpy/ndarraytypes.h:1832,
                 from /opt/python/3.8.2.1/lib/python3.8/site-packages/numpy/core/include/numpy/ndarrayobject.h:12,
                 from /opt/python/3.8.2.1/lib/python3.8/site-packages/numpy/core/include/numpy/arrayobject.h:4,
                 from ctng.cpp:610:
/opt/python/3.8.2.1/lib/python3.8/site-packages/numpy/core/include/numpy/npy_1_7_deprecated_api.h:17:2: warning: #warning "Using deprecated NumPy API, disable it with " "#define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION" [-Wcpp]
 #warning "Using deprecated NumPy API, disable it with " \
  ^~~~~~~
[ 45%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/oc/bksub.c.o
[ 45%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/oc/nrnran123.c.o
[ 45%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/oc/ftime.c.o
[ 45%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/oc/fileio.c.o
[ 45%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/oc/fmenu.c.o
[ 45%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/oc/hoc.c.o
[ 45%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/oc/code2.c.o
[ 46%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/oc/getsym.c.o
[ 46%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/oc/getelm.c.o
[ 46%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/oc/isoc99.c.o
[ 46%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/oc/parallel.c.o
[ 46%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/oc/lineq.c.o
[ 46%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/oc/math.c.o
[ 46%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/oc/version.c.o
[ 47%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/oc/hocedit.c.o
[ 47%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/oc/x.c.o
[ 47%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/memacs/buffer.c.o
[ 47%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/memacs/display.c.o
[ 48%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/oc/nrnfilewrap.c.o
[ 49%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/oc/plt.c.o
[ 49%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/memacs/bind.c.o
[ 49%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/oc/parse.c.o
[ 49%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/oc/subrows.c.o
[ 49%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/oc/symbol.c.o
[ 50%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/memacs/ansi.c.o
[ 50%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/oc/nonlin.c.o
[ 50%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/oc/list.c.o
[ 50%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/memacs/window.c.o
[ 51%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/oc/plot.c.o
[ 51%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/err.c.o
[ 51%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/bdfactor.c.o
[ 51%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/copy.c.o
[ 51%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/oc/scoprand.c.o
[ 52%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/memacs/random.c.o
[ 52%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/oc/ocerf.c.o
[ 53%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/extras.c.o
[ 53%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/memacs/termio.c.o
[ 54%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/oc/xred.c.o
[ 54%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/bkpfacto.c.o
[ 54%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/oc/prmat.c.o
[ 54%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/memacs/basic.c.o
[ 55%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/oc/settext.c.o
[ 55%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/memacs/region.c.o
[ 56%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/oc/functabl.c.o
[ 56%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/memacs/file.c.o
[ 56%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/chfactor.c.o
[ 56%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/memacs/lock.c.o
[ 56%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/lanczos.c.o
[ 56%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/memacs/search.c.o
[ 56%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/memacs/word.c.o
[ 56%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/lufactor.c.o
[ 56%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/dmacheps.c.o
[ 56%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/memacs/fileio.c.o
[ 56%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/givens.c.o
[ 56%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/init.c.o
[ 56%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/oc/regexp.c.o
[ 57%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/ivecop.c.o
[ 58%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/matop.c.o
[ 58%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/iternsym.c.o
[ 58%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/fft.c.o
[ 58%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/iter0.c.o
[ 58%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/memacs/main1.c.o
[ 58%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/memacs/line.c.o
[ 58%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/arnoldi.c.o
[ 58%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/memstat.c.o
[ 58%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/conjgrad.c.o
[ 58%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/memacs/spawn.c.o
[ 58%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/itersym.c.o
[ 59%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/hsehldr.c.o
ctng.cpp:4155:18: warning: 'PyObject* __pyx_f_6neuron_3rxd_10geometry3d_4ctng_tangent_sphere(PyObject*, int)' defined but not used [-Wunused-function]
 static PyObject *__pyx_f_6neuron_3rxd_10geometry3d_4ctng_tangent_sphere(PyObject *__pyx_v_cone, int __pyx_v_whichend) {
                  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
ctng.cpp:3993:15: warning: 'double __pyx_f_6neuron_3rxd_10geometry3d_4ctng_qsolve(double, double, double)' defined but not used [-Wunused-function]
 static double __pyx_f_6neuron_3rxd_10geometry3d_4ctng_qsolve(double __pyx_v_a, double __pyx_v_b, double __pyx_v_c) {
               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
ctng.cpp:3881:18: warning: 'PyObject* __pyx_f_6neuron_3rxd_10geometry3d_4ctng_convert2dto3d(double, double, double, double, double, PyArrayObject*, PyArrayObject*)' defined but not used [-Wunused-function]
 static PyObject *__pyx_f_6neuron_3rxd_10geometry3d_4ctng_convert2dto3d(double __pyx_v_ptx, double __pyx_v_pty, double __pyx_v_x1, double __pyx_v_y1, double __pyx_v_z1, PyArrayObject *__pyx_v_axis, PyArrayObject *__pyx_v_radial_vec) {
                  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
ctng.cpp:2694:18: warning: 'PyObject* __pyx_f_6neuron_3rxd_10geometry3d_4ctng_convert3dto2d(double, double, double, double, double, double, double, double, double, double, double, double)' defined but not used [-Wunused-function]
 static PyObject *__pyx_f_6neuron_3rxd_10geometry3d_4ctng_convert3dto2d(double __pyx_v_x, double __pyx_v_y, double __pyx_v_z, double __pyx_v_px, double __pyx_v_py, double __pyx_v_pz, double __pyx_v_xx, double __pyx_v_xy, double __pyx_v_xz, double __pyx_v_yx, double __pyx_v_yy, double __pyx_v_yz) {
                  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[ 59%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/hessen.c.o
[ 59%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/meminfo.c.o
[ 59%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/memacs/tcap.c.o
[ 59%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/memory.c.o
[ 59%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/matrixio.c.o
[ 60%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/mfunc.c.o
[ 60%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/machine.c.o
[ 60%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/matlab.c.o
[ 60%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/norm.c.o
[ 60%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/otherio.c.o
[ 60%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/pxop.c.o
[ 60%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/qrfactor.c.o
[ 61%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/schur.c.o
[ 61%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/solve.c.o
[ 61%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/sparse.c.o
[ 61%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/sparseio.c.o
[ 61%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/spbkp.c.o
[ 62%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/splufctr.c.o
[ 62%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/spchfctr.c.o
[ 62%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/sprow.c.o
[ 62%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/spswap.c.o
[ 62%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/submat.c.o
/opt/cray/pe/craype/2.7.0/bin/CC -pthread -shared -Wl,-rpath,/opt/python/3.8.2.1/lib build/temp.linux-x86_64-3.8/ctng.o -L/opt/python/3.8.2.1/lib -o /users/bp000347/nrn/build/lib/python/neuron/rxd/geometry3d/ctng.cpython-38-x86_64-linux-gnu.so
[ 63%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/svd.c.o
[ 63%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/symmeig.c.o
[ 63%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/update.c.o
[ 63%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/vecop.c.o
[ 63%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/version.c.o
[ 64%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/zcopy.c.o
[ 64%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/zfunc.c.o
[ 64%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/zgivens.c.o
[ 64%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/zhessen.c.o
[ 64%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/zhsehldr.c.o
[ 65%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/zmachine.c.o
[ 65%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/zlufctr.c.o
[ 65%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/zmatio.c.o
[ 65%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/zmatlab.c.o
[ 65%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/zmatop.c.o
[ 66%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/zmemory.c.o
[ 66%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/znorm.c.o
[ 66%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/zqrfctr.c.o
[ 66%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/zschur.c.o
[ 66%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/zsolve.c.o
[ 67%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/zvecop.c.o
[ 67%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/apcount.c.o
[ 67%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/feature.c.o
[ 67%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/intfire2.c.o
[ 67%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/oclmp.c.o
[ 68%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/ppmark.c.o
[ 68%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/syn.c.o
[ 68%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/exp2syn.c.o
[ 68%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/hh.c.o
[ 69%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/intfire4.c.o
[ 69%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/passive.c.o
[ 69%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/stim.c.o
[ 69%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/vclmp.c.o
[ 69%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/expsyn.c.o
[ 70%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/intfire1.c.o
[ 70%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/netstim.c.o
[ 70%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/pattern.c.o
[ 70%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/svclmp.c.o
[ 70%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/gnu/ACG.cpp.o
[ 71%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/gnu/Binomial.cpp.o
[ 71%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/gnu/Complex.cpp.o
[ 71%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/gnu/DiscUnif.cpp.o
[ 71%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/gnu/Erlang.cpp.o
[ 71%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/gnu/Geom.cpp.o
[ 72%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/gnu/LogNorm.cpp.o
[ 72%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/gnu/HypGeom.cpp.o
[ 72%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/gnu/MLCG.cpp.o
[ 72%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/gnu/NegExp.cpp.o
[ 72%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/gnu/Normal.cpp.o
[ 73%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/gnu/RNG.cpp.o
[ 73%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/gnu/Poisson.cpp.o
[ 73%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/gnu/Random.cpp.o
[ 73%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/gnu/RndInt.cpp.o
[ 73%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/gnu/SmplHist.cpp.o
[ 74%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/gnu/Uniform.cpp.o
[ 74%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/gnu/SmplStat.cpp.o
[ 74%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/gnu/Weibull.cpp.o
[ 74%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/gnu/builtin.cpp.o
[ 74%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/gnu/d_avec.cpp.o
[ 75%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/gnu/d_vec.cpp.o
[ 75%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/abort.c.o
[ 75%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/adams.c.o
[ 75%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/adeuler.c.o
[ 75%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/adrunge.c.o
[ 76%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/boundary.c.o
[ 76%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/advance.c.o
[ 76%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/crank.c.o
[ 76%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/crout.c.o
[ 76%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/csoda.c.o
[ 77%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/deflate.c.o
[ 77%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/csodabnc.c.o
[ 77%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/dimplic.c.o
[ 77%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/scoperf.c.o
[ 77%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/euler.c.o
[ 78%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/exprand.c.o
[ 78%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/expfit.c.o
[ 78%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/f2cmisc.c.o
[ 78%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/factoria.c.o
[ 78%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/force.c.o
[ 79%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/gear.c.o
[ 79%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/gauss.c.o
[ 79%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/getmem.c.o
[ 79%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/harmonic.c.o
[ 79%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/heun.c.o
[ 80%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/hyperbol.c.o
[ 80%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/invert.c.o
[ 80%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/lag.c.o
[ 80%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/legendre.c.o
[ 80%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/newton.c.o
[ 81%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/perpulse.c.o
[ 81%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/normrand.c.o
[ 81%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/perstep.c.o
[ 81%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/poisrand.c.o
[ 81%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/poisson.c.o
[ 82%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/pulse.c.o
[ 82%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/praxis.c.o
[ 82%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/quad.c.o
[ 82%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/ramp.c.o
[ 82%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/revhyper.c.o
[ 83%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/revsawto.c.o
[ 83%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/revsigmo.c.o
[ 83%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/romberg.c.o
[ 83%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/runge.c.o
[ 83%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/sawtooth.c.o
[ 84%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/seidel.c.o
[ 84%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/schedule.c.o
[ 84%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/sigmoid.c.o
[ 84%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/simeq.c.o
[ 84%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/simplex.c.o
[ 85%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/vsparse.c.o
[ 85%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/sparse.c.o
[ 85%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/spline.c.o
[ 85%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/squarewa.c.o
[ 85%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/ssimplic.c.o
[ 86%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/threshol.c.o
[ 86%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/step.c.o
[ 86%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/tridiag.c.o
[ 86%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/sparse_thread.c.o
[ 86%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/newton_thread.c.o
[ 87%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/ssimplic_thread.c.o
[ 87%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/scopmath/crout_thread.c.o
[ 87%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/sparse13/spalloc.c.o
[ 87%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/sparse13/spbuild.c.o
[ 87%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/sparse13/spfactor.c.o
[ 88%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/sparse13/spoutput.c.o
[ 88%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/sparse13/spsolve.c.o
[ 88%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/sparse13/sputils.c.o
[ 88%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/sparse13/cspalloc.c.o
[ 88%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/sparse13/cspbuild.c.o
[ 89%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/sparse13/cspoutput.c.o
[ 89%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/sparse13/cspfactor.c.o
[ 89%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/sparse13/cspsolve.c.o
[ 89%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/sparse13/csputils.c.o
[ 89%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/sundials/cvodes/cvband.c.o
[ 90%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/sundials/cvodes/cvbandpre.c.o
[ 90%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/sundials/cvodes/cvbbdpre.c.o
[ 90%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/sundials/cvodes/cvdense.c.o
[ 90%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/sundials/cvodes/cvdiag.c.o
[ 90%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/sundials/cvodes/cvodea.c.o
[ 91%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/sundials/cvodes/cvodes.c.o
[ 91%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/sundials/cvodes/cvodesio.c.o
[ 91%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/sundials/cvodes/cvspgmr.c.o
[ 91%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/sundials/ida/idaband.c.o
[ 91%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/sundials/ida/idabbdpre.c.o
[ 92%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/sundials/ida/ida.c.o
[ 92%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/sundials/ida/idadense.c.o
[ 92%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/sundials/ida/idaic.c.o
[ 92%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/sundials/ida/idaio.c.o
[ 92%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/sundials/ida/idaspgmr.c.o
[ 93%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/sundials/shared/dense.c.o
[ 93%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/sundials/shared/band.c.o
[ 93%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/sundials/shared/iterative.c.o
[ 93%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/sundials/shared/nvector.c.o
[ 93%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/sundials/shared/nvector_serial.c.o
[ 94%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/sundials/shared/smalldense.c.o
[ 94%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/sundials/shared/spgmr.c.o
[ 94%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/sundials/shared/sundialsmath.c.o
[ 94%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/nvector_nrnparallel_ld.c.o
[ 94%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/sundials/shared/nvector_parallel.c.o
[ 95%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnmpi/nrnmpi.c.o
[ 95%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnmpi/bbsmpipack.c.o
[ 95%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnmpi/mpispike.c.o
[ 95%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnmpi/nrnrt.c.o
[ 95%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnpython/nrnpython.cpp.o
[ 96%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnpython/nrnpy_hoc.cpp.o
[ 96%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnpython/nrnpy_nrn.cpp.o
[ 96%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnpython/nrnpy_p2h.cpp.o
[ 96%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnpython/grids.cpp.o
[ 96%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnpython/rxd.cpp.o
[ 97%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnpython/rxd_intracellular.cpp.o
[ 97%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnpython/rxd_extracellular.cpp.o
[ 97%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnpython/rxd_vol.cpp.o
[ 97%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnpython/rxd_marching_cubes.c.o
[ 97%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnpython/rxd_llgramarea.c.o
[ 98%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivos/listimpl.cpp.o
[ 98%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivos/file.cpp.o
[ 98%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivos/math.cpp.o
[ 98%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivos/memory.cpp.o
[ 98%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivos/string.cpp.o
[ 99%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivos/observe.cpp.o
[ 99%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivos/ustring.cpp.o
[ 99%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivos/regexp.cpp.o
[ 99%] Building CXX object src/nrniv/CMakeFiles/nrniv_lib.dir/__/ivos/resource.cpp.o
building 'neuron.rxd.geometry3d.surfaces' extension
creating build/temp.linux-x86_64-3.8/users
creating build/temp.linux-x86_64-3.8/users/bp000347
creating build/temp.linux-x86_64-3.8/users/bp000347/nrn
creating build/temp.linux-x86_64-3.8/users/bp000347/nrn/src
creating build/temp.linux-x86_64-3.8/users/bp000347/nrn/src/nrnpython
/opt/cray/pe/craype/2.7.0/bin/cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -march=x86-64 -march=x86-64 -fPIC -I/users/bp000347/nrn/share/lib/python/neuron/rxd/geometry3d -I. -I/opt/python/3.8.2.1/lib/python3.8/site-packages/numpy/core/include -I/opt/python/3.8.2.1/include/python3.8 -c surfaces.cpp -o build/temp.linux-x86_64-3.8/surfaces.o -O0
In file included from /opt/python/3.8.2.1/lib/python3.8/site-packages/numpy/core/include/numpy/ndarraytypes.h:1832,
                 from /opt/python/3.8.2.1/lib/python3.8/site-packages/numpy/core/include/numpy/ndarrayobject.h:12,
                 from /opt/python/3.8.2.1/lib/python3.8/site-packages/numpy/core/include/numpy/arrayobject.h:4,
                 from surfaces.cpp:610:
/opt/python/3.8.2.1/lib/python3.8/site-packages/numpy/core/include/numpy/npy_1_7_deprecated_api.h:17:2: warning: #warning "Using deprecated NumPy API, disable it with " "#define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION" [-Wcpp]
 #warning "Using deprecated NumPy API, disable it with " \
  ^~~~~~~
/opt/cray/pe/craype/2.7.0/bin/cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -march=x86-64 -march=x86-64 -fPIC -I/users/bp000347/nrn/share/lib/python/neuron/rxd/geometry3d -I. -I/opt/python/3.8.2.1/lib/python3.8/site-packages/numpy/core/include -I/opt/python/3.8.2.1/include/python3.8 -c /users/bp000347/nrn/src/nrnpython/rxd_marching_cubes.c -o build/temp.linux-x86_64-3.8/users/bp000347/nrn/src/nrnpython/rxd_marching_cubes.o -O0
/opt/cray/pe/craype/2.7.0/bin/cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -march=x86-64 -march=x86-64 -fPIC -I/users/bp000347/nrn/share/lib/python/neuron/rxd/geometry3d -I. -I/opt/python/3.8.2.1/lib/python3.8/site-packages/numpy/core/include -I/opt/python/3.8.2.1/include/python3.8 -c /users/bp000347/nrn/src/nrnpython/rxd_llgramarea.c -o build/temp.linux-x86_64-3.8/users/bp000347/nrn/src/nrnpython/rxd_llgramarea.o -O0
/opt/cray/pe/craype/2.7.0/bin/CC -pthread -shared -Wl,-rpath,/opt/python/3.8.2.1/lib build/temp.linux-x86_64-3.8/surfaces.o build/temp.linux-x86_64-3.8/users/bp000347/nrn/src/nrnpython/rxd_marching_cubes.o build/temp.linux-x86_64-3.8/users/bp000347/nrn/src/nrnpython/rxd_llgramarea.o -L/opt/python/3.8.2.1/lib -o /users/bp000347/nrn/build/lib/python/neuron/rxd/geometry3d/surfaces.cpython-38-x86_64-linux-gnu.so
[ 99%] Built target rx3dextensions_0
Scanning dependencies of target rx3d
[ 99%] Built target rx3d
/users/bp000347/nrn/src/nrnpython/nrnpy_hoc.cpp: In function 'PyObject* iternext_sl(PyHocObject*, hoc_Item*)':
/users/bp000347/nrn/src/nrnpython/nrnpy_hoc.cpp:1733:1: warning: control reaches end of non-void function [-Wreturn-type]
 }
 ^
[ 99%] Linking CXX static library ../../lib/libnrniv.a
[ 99%] Built target nrniv_lib
Scanning dependencies of target nrniv
Scanning dependencies of target hoc_module
[ 99%] Generating inithoc.cpp
[ 99%] Building CXX object src/nrniv/CMakeFiles/nrniv.dir/__/ivoc/nrnmain.cpp.o
[ 99%] Building C object src/nrniv/CMakeFiles/nrniv.dir/__/oc/ockludge.c.o
[100%] Building C object src/nrniv/CMakeFiles/nrniv.dir/__/oc/modlreg.c.o
[100%] Linking CXX executable ../../bin/nrniv
/usr/bin/ld: cannot find -lpython3.8
collect2: error: ld returned 1 exit status
make[2]: *** [src/nrniv/CMakeFiles/nrniv.dir/build.make:152: bin/nrniv] Error 1
make[1]: *** [CMakeFiles/Makefile2:391: src/nrniv/CMakeFiles/nrniv.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
Building python module with: /opt/python/3.8.2.1/bin/python
[100%] Built target hoc_module
make: *** [Makefile:130: all] Error 2
[  1%] Built target readline
[  6%] Built target nocmodl
[ 10%] Built target modlunit
[ 10%] Built target nrnversion_h
[ 10%] Update hh.mod for CoreNEURON compatibility
[ 10%] Built target hh_update
[ 97%] Built target nrniv_lib
[ 97%] Linking CXX executable ../../bin/nrniv
/usr/bin/ld: cannot find -lpython3.8
collect2: error: ld returned 1 exit status
make[2]: *** [src/nrniv/CMakeFiles/nrniv.dir/build.make:152: bin/nrniv] Error 1
make[1]: *** [CMakeFiles/Makefile2:391: src/nrniv/CMakeFiles/nrniv.dir/all] Error 2
make: *** [Makefile:130: all] Error 2
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -109,6 +109,11 @@ set(NRN_RX3D_OPT_LEVEL
     CACHE STRING "Optimization level for Cython generated files (non-zero may compile slowly)")
 
 # =============================================================================
+# Set Python additional versions earlier (especially for old CMake)
+# =============================================================================
+set(Python_ADDITIONAL_VERSIONS 3 3.9 3.8 3.7 3.6 3.5 3.4 3.3 3.2 3.1 3.0 2.8 2.7 2.6)
+
+# =============================================================================
 # Include cmake modules
 # =============================================================================
 list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake/modules)
@@ -255,7 +260,6 @@ endif()
 # =============================================================================
 # Enable Python support
 # =============================================================================
-set(Python_ADDITIONAL_VERSIONS 3 3.9 3.8 3.7 3.6 3.5 3.4 3.3 2.8 2.7)
 find_package(PythonInterp REQUIRED)
 if(NRN_ENABLE_PYTHON)
   find_package(PythonLibsNew ${PYTHON_VERSION_MAJOR} REQUIRED)

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -255,6 +255,7 @@ endif()
 # =============================================================================
 # Enable Python support
 # =============================================================================
+set(Python_ADDITIONAL_VERSIONS 3 3.9 3.8 3.7 3.6 3.5 3.4 3.3 2.8 2.7)
 find_package(PythonInterp REQUIRED)
 if(NRN_ENABLE_PYTHON)
   find_package(PythonLibsNew ${PYTHON_VERSION_MAJOR} REQUIRED)

--- a/docs/cmake_doc/options.rst
+++ b/docs/cmake_doc/options.rst
@@ -217,8 +217,8 @@ NRN_PYTHON_DYNAMIC:STRING=
 
 PYTHON_EXECUTABLE:PATH=
 -----------------------
-  Use this python instead of the python cmake finds by default.
-  Must be a full path. I generally use
+  Use provided python binary instead of the one found by CMake.
+  This must be a full path. We generally use
 
   .. code-block:: shell
 

